client/post-editor/media-modal/secondary-actions.jsx

```diff
@@ -96,89 +78,18 @@ const MediaModalSecondaryActions = React.createClass( {
 		return buttons;
 	},
 
-	toggleMobilePopover() {
-		this.setState( {
-			isMobilePopoverVisible: ! this.state.isMobilePopoverVisible
-		} );
-	},
-
-	renderMobileButtons() {
-		const buttons = this.getButtons();
-
-		if ( ! buttons.length ) {
-			return;
-		}
-
-		const classes = classNames( 'editor-media-modal__secondary-action', 'button', 'is-mobile', 'is-link', {
-			'is-active': this.state.isMobilePopoverVisible
-		} );
-
-		const menuItems = buttons.map( ( button ) => {
-			const onClick = () => {
-				this.toggleMobilePopover();
-
-				if ( button.onClick ) {
-					button.onClick();
-				}
-			};
-
-			return React.createElement( PopoverMenuItem, {
-				key: button.key,
-				action: button.key,
-				onClick: onClick
-			}, button.value );
-		} );
-
-		return (
-			<button
-				ref={ this.setMobilePopoverContext }
-				onClick={ this.toggleMobilePopover }
-				className={ classes }>
-				<span className="screen-reader-text">{ this.translate( 'More Options' ) }</span>
-				<Gridicon icon="ellipsis" size={ 24 } />
-				<PopoverMenu
-					context={ this.state.mobilePopoverContext }
-					isVisible={ this.state.isMobilePopoverVisible }
-					onClose={ this.toggleMobilePopover }
-					position="top right"
-					className="popover is-dialog-visible">
-					{ menuItems }
-				</PopoverMenu>
-			</button>
-		);
-	},
-
-	renderDesktopButtons() {
-		return this.getButtons().map( ( button ) => {
-			return React.createElement( 'input', Object.assign( {
-				type: 'button'
-			}, button, {
-				className: classNames( 'editor-media-modal__secondary-action', 'button', 'is-desktop', button.className )
-			} ) );
-		} );
-	},
-
-	renderPlanStorage() {
```

Comment:
I believe we'll need this 😄  As is we won't ever see this component. Let's wait for #11639 to land, and we can rebase both this branch and the target branch.